README.md

```diff
@@ -1,4 +1,4 @@
-# Marlin 3D Printer Firmware
+﻿# Marlin 3D Printer Firmware
```

Comment:
Be sure to use "UTF-8" encoding, not "UTF-8 with BOM" encoding. Otherwise the first line of text and source files will keep showing up as changed.